internal/distributed/indexnode/service.go

```diff
@@ -289,6 +289,18 @@ func (s *Server) GetMetrics(ctx context.Context, request *milvuspb.GetMetricsReq
 	return s.indexnode.GetMetrics(ctx, request)
 }
 
+func (s *Server) Analysis(ctx context.Context, request *indexpb.AnalysisRequest) (*commonpb.Status, error) {
```

Comment:
use CreateJob and it should support both AnalysisRequest and IndexRequest, something like
message CreateJobRequestV2 {
   AnalysisRequest
   IndexRequest
}